docs/upgradeVotingKeys.md

```diff
@@ -0,0 +1,50 @@
+`symbol-bootstrap upgradeVotingKeys`
+====================================
+
+It upgrades the voting keys and files when required.
+
+Voting file upgrade:
+- If the node's current voting file has an end epoch close to the current epoch ("close to expiring") this command creates a new 'private_key_treeX.dat' that continues the current file.
+- "Close to expiring" happens when the epoch is in the upper half of the voting file. If the file's epoch length is 720, close to expiring will be 360+.
```

Comment:
I think at this stage allowing the user to upgrade 3 months in advance is fine, especially because this is the first upgrade and people need to get the process in place (like all nodes). We could reduce the default values in a later voting key upgrade

Reply:
I think a month is more than enough time to run the update command.
Allnode could create a cron job that runs every 6 months to generate new key files.  If they want to run it early, then nothing stopping them since there is an override option.
We are kind thinking similar except your suggestion would require an update later to the bootstrap.

Reply:
Ok, I will upgrade the preset. Defaults will be:

votingKeyDesiredLifetime: 6 months in epochs (how long the file last)
votingKeyDesiredFutureLifetime: 1 month in epochs (1 month before the current file expires bootstrap will create a new file)